lib/klp.pm

```diff
@@ -45,10 +46,13 @@ sub install_klp_product {
         $lp_module  = 'SLE-Live-Patching';
     }
 
-    #install kgraft product
-    zypper_call("ar http://download.suse.de/ibs/SUSE/Products/$lp_module/$version/$arch/product/ kgraft-pool");
-    zypper_call("ar $release_override http://download.suse.de/ibs/SUSE/Updates/$lp_module/$version/$arch/update/ kgraft-update");
-    zypper_call("ref");
+    if ($livepatch_repo) {
+        zypper_ar("$utils::OPENQA_FTP_URL/$livepatch_repo", name => "repo-live-patching");
+    }
+
+    # install kgraft product
+    zypper_ar("http://download.suse.de/ibs/SUSE/Products/$lp_module/$version/$arch/product/",                 name => "kgraft-pool");
+    zypper_ar("$release_override http://download.suse.de/ibs/SUSE/Updates/$lp_module/$version/$arch/update/", name => "kgraft-update");
```

Comment:
Was the `zypper_call("ref");` from line 51 above dropped intentionally? Otherwise LGTM.

Reply:
@mdoucha Yes, because `zypper_ar()` does also refresh (I wrote that code in https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/utils.pm#L610 to simplify adding repos, it would fail without it).

See http://quasar.suse.cz/tests/5450/file/serial_terminal.txt
```
# zypper -n --gpg-auto-import-keys ref --repo repo-live-patching | cat; ( exit ${PIPESTATUS[0]} ); echo UcdbC-$?-
# zypper -n --gpg-auto-import-keys ref --repo kgraft-pool | cat; ( exit ${PIPESTATUS[0]} ); echo B4wKu-$?-
# zypper -n --gpg-auto-import-keys ref --repo kgraft-update | cat; ( exit ${PIPESTATUS[0]} ); echo Opqy2-$?-
```